packages/waffle-chart/src/CdcWaffleChart.tsx

```diff
@@ -338,7 +319,7 @@ const WaffleChart = ({ config, isEditor, link }) => {
   )
 }
 
-const CdcWaffleChart = ({ configUrl, config: configObj, isDashboard = false, isEditor = false, setConfig: setParentConfig }) => {
+const CdcWaffleChart = ({ configUrl, config: configObj, isDashboard = false, isEditor = false, setConfig: setParentConfig }: CdcWaffleChartTypes): React.ReactComponentElement => {
```

Comment:
I have a change in another PR which fixes the need to define this at the end: https://github.com/CDCgov/cdc-open-viz/pull/891/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0